### DISCUSSION TITLE
App closing animations

### DISCUSSION THREAD

**Discussion by widkit** [LawnchairLauncher/lawnchair#3110] [Ideas]
With the latest Niagara Launcher beta, you CAN use those effects without root or anything. My previous suggestion was denied saying it's impossible even if it was not.

**Comment by tropicaaal** [comment]
relevant - https://github.com/LawnchairLauncher/lawnchair/issues/2856#issuecomment-1236268578

**Comment by SuperDragonXD** [comment]
Hello! I know that this reply is a bit late, but this should now be supported on Lawnchair 14 and up.
